client/src/pages/DemoUser/useStyles.ts

```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  submit: {
+    margin: theme.spacing(3, 2, 2),
+    padding: 10,
+    width: 160,
+    height: 56,
+    borderRadius: theme.shape.borderRadius,
+    marginTop: 49,
+    fontSize: 16,
+    backgroundColor: '#f14140',
```

Comment:
We shouldn't need this, we can instead leverage the material ui [theme](https://v4.mui.com/customization/theming/). Our theme has the same [primary color](https://v4.mui.com/customization/palette/#palette) (`#f14140`) and the `color="primary"` prop should take care of the rest 👍🏼

When using material ui, we generally want to avoid individual styling of components whenever possible and instead think about forming reusable component variations (e.g. instead of 6 `TextFields` in a form with the same styles applied to each individually, we can create a reusable `FormTextField` component). [This reddit answer](https://www.reddit.com/r/reactjs/comments/os126m/comment/h6m15t2/?utm_source=share&utm_medium=web2x&context=3) puts it quite well:
> If you are always making the same adjustments to the material-ui components you should look into theming (eg. give every primary button the color of your chosing, or change spacing). If you use a library like material-ui you should try to avoid styling every implementation of a component separately. Only try to style when absolutely necessary, or make your own abstraction on top of a material ui component, and import those in other components to limit styling (eg, a Confirm or Cancel button, or a Confirmation modal etc). 

We should be able to simply remove the `backgroundColor` style and have the same results so for now let's do that. The rest is just to give you a better understanding of material ui for your future use cases. Give it a try and let me know! 😇